### PR TITLE
Initiating release of 0.11.0-m2

### PR DESCRIPTION
**Pull request by mkolesnik**
Initiating release of 0.11.0-m2

### CODE REVIEW

File: releases/v0.11.0-m2.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+version: v0.11.0-m2
+name: 0.11.0-m2
+pre-release: true
+status: shipyard
+components:
+  shipyard: 124b5372115351a9df8ffc9fa62c1b9c510873e4
```
Comment (tpantelis) [inline releases/v0.11.0-m2.yaml:7]: This points to a much older commit:

```
commit 124b5372115351a9df8ffc9fa62c1b9c510873e4
Author: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Date:   Wed Jul 7 17:08:18 2021 +0000

    Bump github.com/onsi/gomega from 1.13.0 to 1.14.0

```

Don't we want the latest commit?

```suggestion
  shipyard: 60a7326c0708cdcd83d2925e491de3e62af0f90c
```

**Review by tpantelis — CHANGES_REQUESTED** [review]

File: releases/v0.11.0-m2.yaml (hunk 1 above)
Comment (mkolesnik) [reply releases/v0.11.0-m2.yaml:7]: Ah right :roll_eyes: it looks at my forks not the upstream, need to change that..

Comment (mkolesnik) [reply releases/v0.11.0-m2.yaml:7]: I opened this issue - https://github.com/submariner-io/releases/issues/211

**Comment by mkolesnik** [comment]
Superceded by #214

**Pull request closed without merging**